modules/002-deckhouse/monitoring/prometheus-rules/migration-alerts.yaml

```diff
@@ -0,0 +1,16 @@
+- name: d8.migration-alerts
```

Comment:
Please move it to https://github.com/deckhouse/deckhouse/tree/main/modules/340-monitoring-deckhouse module.

it will not work because deckhouse module deploy before prometheus crd's

Reply:
moved